jenkins_jobs/sample_test_jobs/docker/sample_tests_beryllium/sample_tests_beryllium.yaml

```diff
@@ -0,0 +1,412 @@
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.

```

Comment:
answering to myself to 1:
- we need separate conf files for docker-based tests mainly because of the container IP addressing scheme... is there any chance that we could deal with this (i.e. not having to replicate the conf files for docker, just to change the nodes IP addresses?)